public/theme/extensions-2021/source/scss/_typography.scss

```diff
@@ -0,0 +1,8 @@
+h1, h2, h3, h4, h5, h6 {
```

Comment:
(OK, let's say that all the scss files can go)

Reply:
Ok, deleted